docs/source/canopen.rst

```diff
@@ -100,7 +100,7 @@ parts. First is the C++ class which represents the node as a whole including
 the object dictionary. Second is the main file which setups CANopen and
 processes incoming data.
 
-Lets first look at the Object Dictionary contained in ``TestCanNode.hpp``.
+Lets first look at the Object Dictionary contained in ``RPDOCanNode.hpp``.
 
 .. code-block:: C++
```

Comment:
This is probably a good place to document the new architecture. Can you update this to explain how to use the new setup? Or would you rather document it on the website?

I only want it documented in one place, so we can have a single master of record. I think it makes more sense to be here and get put on ReadTheDocs, but our ReadTheDocs isn't set up very well right now, so nobody uses it, meaning the website might be better.

Regardless of what you decide, this example should be updated or deleted. Don't update this PR, though, just put that in the new one because that's the one with the new macros